src/daisy.js

```diff
@@ -132,12 +149,60 @@ var daisy = (function(Object, Array, Math, undefined){
 	var Computation = function(initParam, options){
 		this.currentVal = initParam + '';
 		this.options = options || {};
+		if (this.options.round && !this.options.format) {
+			this.options.format = 'n';
+		}
+		if (this.options.INR && this.options.commas === void 0) {
+			this.options.commas = true;
+		}
+	};
+
+	Computation.prototype.format = function () {
+		var formatted,
+			placeValue = 1e-2,
+			decimalPlaces = 0,
+			format = this.options.format || 'n.nn',
+			operation = {
+				"up": "ceil",
+				"down": "floor",
+				"round": "round"
+			}[this.options.round] || 'round';
+
+		// check for valid format string, otherwise use default
+		if (
+			format.match(/\./g) && format.match(/\./g).length > 1 ||
+			/[^n\.0\$]/i.test(format)
+		){
+			format = 'n.nn';
+		}
+
+		// not actually replacing anything, using the callback function
+		// to extract information from the regex match.
+		format.replace(/n(0*)\.?(n*)/ig, function (m, aboveOne, belowOne) {
+			if (aboveOne || belowOne) {
+				var isDecimal = !aboveOne && !!belowOne;
+				decimalPlaces = aboveOne ? 0 : belowOne.length || 0;
+				placeValue = Math.pow(10, isDecimal ? -decimalPlaces : aboveOne.length);
+			} else {
+				placeValue = 1;
+			}
+		});
+
+		// round to desired place value
+		formatted = (Math[operation](+this.currentVal / placeValue) * placeValue).toFixed(decimalPlaces);
+
+		return makeNumberString({
+			"formatted": formatted,
+			"hasDollarSign": /\$/.test(format),
+			"decimalPlaces": decimalPlaces,
+			"commas": this.options.commas,
+			"locale": this.options.INR ? 'hi-IN' : 'en-US'

```

Comment:
Sets the locale to 'hindi-india,' which allows the user to use lakh/crore comma groupings.